src/Npgsql/NpgsqlOperationInProgressException.cs

```diff
@@ -1,14 +1,18 @@
-﻿using System;
-
-namespace Npgsql
+﻿namespace Npgsql
 {
     /// <summary>
     /// Thrown when trying to use a connection that is already busy performing some other operation.
     /// Provides information on the already-executing operation to help with debugging.
     /// </summary>
-    public sealed class NpgsqlOperationInProgressException : InvalidOperationException
+    public sealed class NpgsqlOperationInProgressException : NpgsqlException
     {
-        internal NpgsqlOperationInProgressException(NpgsqlCommand command)
+        /// <summary>
+        /// Creates a new instance of <see cref="NpgsqlOperationInProgressException" />.
+        /// </summary>
+        /// <param name="command">
+        /// A command which was in progress when the operation which triggered this exception was executed.
+        /// </param>
+        public NpgsqlOperationInProgressException(NpgsqlCommand command)
```

Comment:
Would be better to pass the text as parameter instead of the command because it losses dependencies and makes code more test friendly.

Reply:
But in real life the command is sometimes useful because it references the parameters... Let's see if people make requests about this exception we can always improve it.